crates/base/src/vector/svecf32.rs

```diff
@@ -171,6 +171,11 @@ impl<'a> VectorBorrowed for SVecf32Borrowed<'a> {
         }
     }
 
+    #[inline(always)]
+    fn to_index_vec(&self) -> Vec<(u32, Self::Scalar)> {
+        std::iter::zip(self.indexes.to_vec(), self.values.to_vec()).collect()
+    }
```

Comment:
This function should be moved to `OperatorInverted` in `./crates/inverted/operator.rs`. The implementation for all other types should be `unimplemented!()`.

Reply:
fixed